packages/bpk-component-split-input/src/BpkInputField.js

```diff
@@ -62,6 +63,7 @@ BpkInputField.propTypes = {
   value: PropTypes.string,
   focus: PropTypes.bool.isRequired,
   index: PropTypes.number.isRequired,
+  name: PropTypes.string,
```

Comment:
As this is not marked as required it will need a default value otherwise the value when rendered will be blank and/or crrash

Reply:
Thanks for pointing that out!